src/game.h

```diff
@@ -893,6 +893,7 @@ class game
         pimpl<stats_tracker> stats_tracker_ptr;
         pimpl<kill_tracker> kill_tracker_ptr;
         pimpl<memorial_logger> memorial_logger_ptr;
+        pimpl<spell_events> spell_events_ptr;
```

Comment:
I feel like this might have fitted better as a member of `avatar`, rather than `game`.